tests/cli/test_cmds_spark_run.py

```diff
@@ -393,6 +393,7 @@ def test_configure_and_run_docker_container(
         args.dry_run = True
         args.mrjob = False
         args.nvidia = False
+        args.cluster_manager = "mesos"
```

Comment:
Nit: For consistency, maybe use CLUSTER_MANAGER_MESOS? 
Same as everywhere else.